shared/js/ui/models/whitelist.es6.js

```diff
@@ -25,6 +25,14 @@ Whitelist.prototype = window.$.extend({},
                     value: false
                 }
             })
+            // Remove domain whitelist opt-in status, if present
+            this.fetch({
```

Comment:
We're storing opt-in settings per domain. For instance, say I'm on `example.com/index.html` and I whitelist the site and fill out the form, then I navigate to `example.com/help.html` and decide the site isn't actually broken and turn privacy protection back on. The domain and path will be attached to the ept.on pixel, because I previously opted in to sharing breakage data on this domain.